File: flutter/tasc_travel_app/lib/models/place.dart
class Place {
  final int id;
  final String name;
  final double rating;
  final String imageUrl;

  Place({required this.id, required this.name, required this.rating, required this.imageUrl});

  factory Place.fromJson(Map<String, dynamic> json) {
    return Place(
      id: json['id'],
      name: json['name'],
      rating: json['rating'],
      imageUrl: json['imageUrl'],
    );
  }
}

File: flutter/adfa_api/src/main/java/org/example/adfa_api/Place.java
package org.example.adfa_api;

public class Place {
    private Long id;
    private String name;
    private double rating;
    private String imageUrl;

    // Tạo Constructor, Getters và Setters
    public Place(Long id, String name, double rating, String imageUrl) {
        this.id = id;
        this.name = name;
        this.rating = rating;
        this.imageUrl = imageUrl;
    }

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public double getRating() {
        return rating;
    }

    public String getImageUrl() {
        return imageUrl;
    }
}